fluent/test/errors_test.js

```diff
@@ -0,0 +1,96 @@
+"use strict";
+
+import assert from "assert";
+import ftl from "@fluent/dedent";
+
+import FluentBundle from "../src/bundle";
+
+suite("Errors", function() {
```

Comment:
I know that you like your tests verbose, but this test file scared me.

I think one suite with two tests would provide more value to readers, and give the same test coverage. "Two calls" w/ `errs`, and "First error is thrown" w/out.

Reply:
You're right, I'll make this shorter.